internal/postgres/config.go

```diff
@@ -425,3 +426,29 @@ chmod +x /tmp/pg_rewind_tde.sh
 
 	return append([]string{"bash", "-ceu", "--", script, "startup"}, args...)
 }
+
+// ReleaseIsFinal returns whether or not t is definitively past the final
+// scheduled release of a Postgres version.
+// See: https://www.postgresql.org/support/versioning/
+func ReleaseIsFinal(majorVersion int, t time.Time) bool {
```

Comment:
⛏️ This doesn't strike me as configuration. How about putting it in a new file, `internal/postgres/versions.go`?